packages/nimble-blazor/NimbleBlazor/Components/NimbleTabs.razor

```diff
@@ -1,7 +1,7 @@
 @namespace NimbleBlazor
 <nimble-tabs
     activeid="@BindConverter.FormatValue(ActiveId)"
-    @onnimbletabsactiveidchange="(__value) => UpdateActiveId(__value.ActiveId)"
+    @onnimbletabsactiveidchange="(__value) => UpdateActiveId(__value?.ActiveId)"
```

Comment:
Note: I considered changing this to pass in the entire arguments object instead of just one field but ultimately determined that was a bigger change that would require a larger discussion.